### PR TITLE
Bug 2037237: Add "This is a CD-ROM boot source" checkbox to customize wizard

### PR DESCRIPTION
**Pull request by avivtur**
**Fixes**:
https://bugzilla.redhat.com/show_bug.cgi?id=2037237

**Solution Description**:
Exposing a checkbox in advanced wizard, to define the boot source as a CD-ROM boot source.

**Screen shots / Gifs for design review**:
**After**:

https://user-images.githubusercontent.com/67270715/150340879-4632724b-8a16-4af8-a676-c181e73b24be.mp4

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

**Comment by avivtur** [comment]
@vojtechszocs @glekner @yaacov @metalice @pcbailey @hstastna @upalatucci please review

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/CDRomBootSourceCheckbox.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { FormField, FormFieldType } from '../../form/form-field';
+import { FormFieldRow } from '../../form/form-field-row';
+import { VMSettingsField } from '../../types';
+import { getFieldId } from '../../utils/renderable-field-utils';
+
+export const CDRomBootSourceCheckbox: React.FC<CDRomBootSourceCheckboxProps> = React.memo(
```
Comment (metalice) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/CDRomBootSourceCheckbox.tsx:9]: u really need react.memo here?

Comment (avivtur) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/CDRomBootSourceCheckbox.tsx]: removed thanks :)

**Pull request merged**